lib/elixir/lib/integer.ex

```diff
@@ -6,28 +6,37 @@ defmodule Integer do
   import Bitwise
 
   @doc """
-  Determines if an integer is odd.
+  Determines if `integer` is odd.
 
-  Returns `true` if `n` is an odd number, otherwise `false`.
+  Returns `true` if the given `integer` is an odd number,
+  otherwise it returns `false`.
 
   Allowed in guard clauses.
 
   ## Examples
 
-      iex> Integer.is_odd(3)
+      iex> Integer.is_odd(5)
       true
 
-      iex> Integer.is_odd(4)
+      iex> Integer.is_odd(6)
```

Comment:
because we are already testing in the tests is_odd(3), so I thought it was better to change it in the docs for
> is_odd(5) and is_odd(6)

than leave it in descending order
is_odd(5) and is_odd(4)